README.md

```diff
@@ -1,34 +1,63 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# SILACAK
+
+A progressive web app (PWA) to track COVID19 cases.
+
+## Purpose and Principles
+
+Help field officer (TNI) to easily tracing the cases. According to WHO Tracing ratio should be 1:15.
+
+### Principles
+
+- Working without reliable connection (even offline mode)
+- New user can input the data easily.
+- Performance
+- Secure
```

Comment:
For this, we also need to further elaborate on these points, just like what we did on [wargabantuwarga.com](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/README.md#principles)

We can also copy those principles over here. What do you think, @zainfathoni?

Reply:
Agreed. However, copying over the content potentially causes them to be out-of-sync when we're updating one of them.
Therefore, please add a link to [the **Principles** section at the original document](https://github.com/kawalcovid19/wargabantuwarga.com#principles) instead.